docs/en/about-appium/getting-started.md

```diff
@@ -184,7 +185,7 @@ So here is how we begin to construct a session in our test file:
 ```js
 // javascript
 const opts = {
-  path: '',
+  path: '/wd/hub',
```

Comment:
do we want to update this to `''` for appium 2?

Reply:
Probably yes.
We should update the sample code as well, so I think it will be a good time to update here and the sample then.
(thus, not at this time)